app/Nova/DuesTransaction.php

```diff
@@ -65,10 +65,10 @@ public function fields(Request $request): array
             ID::make()
                 ->sortable(),
 
-            BelongsTo::make('Paid By', 'user', 'App\\Nova\\User')
+            BelongsTo::make('Paid By', 'user', User::class)
                 ->searchable(),
 
-            BelongsTo::make('Dues Package', 'package', 'App\\Nova\\DuesPackage'),
+            BelongsTo::make('Dues Package', 'package', DuesPackage::class),
```

Comment:
Since App\x and App\Nova\x are different, we need to make sure this is referencing the right one

Reply:
each of those files is in the `\App\Nova` namespace so if you say `Payment` that means `\App\Nova\Payment`

Reply:
each of those files is in the `\App\Nova` namespace so if you say `Payment` that means `\App\Nova\Payment`